src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  align-items: center;
+  padding: $calendar-padding;
+  gap: $cell-gap;
+  flex-wrap: wrap;
+  max-width: ($cell-side-size + $cell-gap)
+    * $columns-amount + $calendar-padding * 2;
+
+  @each $weeknum, $weekday in $weekdays {
+    &--start-day--#{$weekday} &__date:first-child {
+      margin-left: ($cell-side-size + $cell-gap) * $weeknum;
+    }
+  }
+
+  @for $month-length from 28 through 31 {
+    $month-index: $month-length + 1;
+    &--month-length--#{$month-length} &__date:nth-child(n + #{$month-index}) {
+      display: none;
+    }
+  }
+
+  &__date {
+    @include cell($cell-side-size);
+
+    background-color: #eee;
+    border: 1px solid black;
+    text-align: center;
+    font-size: 30px;
+    line-height: $cell-side-size;
```

Comment:
I belive there is no need to set this parameter

Reply:
fixed👍